src/text/skeleton.scss

```diff
@@ -3,23 +3,29 @@
 @import '../style/index';
 
 @mixin iui-text-skeleton {
-  display: inline-block;
+  width: fit-content;
```

Comment:
Why changed?

Reply:
`inline-block` took full width, `fit-content` only goes the width of the text it is covering up.

Reply:
I don't think we can use some text for skeletons because when we are showing skeleton, we are waiting for some content to be downloaded so for me it doesn't make sense to add some random text there to show skeleton, also it should break my skeleton menu item because:
![image](https://user-images.githubusercontent.com/36186912/163322248-060582d3-607b-4341-a92b-3fe8abf31e6c.png)

Reply:
Removed `width: fit-content;`.  Did not replace `display: inline-block;` because I was thinking it makes sense to keep the `display` set to whatever it should be when skeleton is not applied.

Reply:
I also added an additional demo, a `<p>` with no content inside to replicate what you have going on in menu.

Reply:
Removed `width: fit-content;` again.